### DISCUSSION TITLE
项目打包问题

### DISCUSSION THREAD

**Discussion by PureDimension** [liusy58/NJU_AdvancedProgramming_SP22#186] [Q&A] [answered]
我用vs搭载qt插件写的项目，想问一下，打包的时候是不是只要有以下文件，在别人的电脑上就可以完成编译？也就是说是不是提交这些就可以？（没用.ui文件）
![image](https://user-images.githubusercontent.com/100058437/165673031-6009f982-b223-48e8-8a80-5a5683e0fc69.png)

**Comment by liusy58** [comment, marked as answer]
可以用室友电脑跑一下。

**Reply by PureDimension** [reply]
最后决定把.sln和.pro一起打包交上去了hmm
